### ISSUE TITLE
Feature Request: @versions -> @Entity({ auditing: true })

### ISSUE THREAD

**Issue by davidsparkles** [fullstack-build/fullstack-one#92]
**Package**

- db
- schema-builder

**Description / Use Case for a Feature**

Make the old `@version` directive work again. Or more general, provide a way to provide versioning on tables, e.g. using JSON diff.

**Comment by davidsparkles** [comment]
The following error is thrown, when inserting an item into a Table with versioning. The error is thrown within the `_meta.create_version()` trigger function. Weirdly this happens only on my local postgres but not on aiven. Maybe, I need to use an older version of plv8.
- Aivens plv8 version: 1.4.4
- Local plv8 version: 1.4.8

```sh
error: { error: buffer 1078 is not owned by resource owner TopTransaction
    at Connection.parseE (/Users/davidthol/space/fullstack-one/packages/db/node_modules/pg/lib/connection.js:554:11)
    at Connection.parseMessage (/Users/davidthol/space/fullstack-one/packages/db/node_modules/pg/lib/connection.js:379:19)
    at Socket.<anonymous> (/Users/davidthol/space/fullstack-one/packages/db/node_modules/pg/lib/connection.js:119:22)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
  name: 'error',
  length: 129,
  severity: 'ERROR',
  code: 'XX000',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'resowner.c',
  line: '875',
  routine: 'ResourceOwnerForgetBuffer' }
```

**Comment by terehov** [comment]
@davidsparkles As mentioned on the phone earlier today, I would rather remove plv8 and therefore this diffing altogether and only store the previous and the current state as a full serialized object.

**Comment by davidsparkles** [comment]
solved in #101
